### PR TITLE
Upgrade for Node 6 compatibility

### LINKED ISSUE #455: Warnings with Node 6
When running a steal-tools build the following warnings (among others) are shown:

```
(node:14137) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
(node) v8::ObjectTemplate::Set() with non-primitive values is deprecated
(node) and will stop working in the next major release.
```

### PR DESCRIPTION
**Pull request by matthewp**
Fixes #455

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "grunt-cli": "^0.1.13",
     "grunt-contrib-concat": "~0.3.0",
     "grunt-contrib-copy": "~0.5.0",
-    "grunt-contrib-jshint": "^0.10.0",
+    "grunt-contrib-jshint": "^1.0.0",
```
Comment (daffl) [inline package.json]: Can we get rid of Grunt the same way we did for all the other modules?

Comment (matthewp) [inline package.json]: I don't think so, we have a grunt task we use to test our grunt tasks.

Comment (matthewp) [inline package.json]: We could maybe move the grunt tasks to a different repo/npm though.

Comment (daffl) [inline package.json]: Keeping it to test the task is fine. I was just thinking about JSHint, Testee and the release task.

Comment (matthewp) [inline package.json]: Yeah, we can do this at any time.

**Pull request merged**